scss/components/_title-bar.scss

```diff
@@ -42,7 +42,7 @@ $titlebar-icon-spacing: 0.25rem !default;
 
     @if $global-flexbox {
       display: flex;
-      justify-content: space-between;
+      justify-content: flex-start;
```

Comment:
Ok I see when you dont use title bar left or right then flex start makes sense. I've added to your test case to demonstrate.